components/needs_team/needs_team.tsx

```diff
@@ -139,6 +139,9 @@ export default class NeedsTeam extends React.PureComponent<Props, State> {
 
     public componentDidMount() {
         startPeriodicStatusUpdates();
+        if (this.state.team) {
+            this.initTeam(this.state.team);
```

Comment:
So, the important change in this file is that we're now calling this on mount instead of in the constructor, is that right?

Reply:
good thing is this file is now replaced with team_controller 😛 in my upcoming pr

Reply:
I'd say to remember to fix this there as well then, but I guess it's not even possible to do it in the constructor since it's becoming a functional component